bridges/otellogr/logsink.go

```diff
@@ -82,6 +82,10 @@ func newConfig(options []Option) config {
 		c.provider = global.GetLoggerProvider()
 	}
 
+	if c.ctx == nil {
```

Comment:
This will always be nil since there's no way to pass the config. So we can remove the config attribute and pass `context.Background()` directly here.